src/app/record-ide/components/record-ide.component.ts

```diff
@@ -171,7 +171,7 @@ export class RecordIDEComponent implements OnInit {
   }
 
   navigateToLibraries() {
-    window.open("https://github.com/code-kern-ai/record-ide/blob/dev/requirements.txt", "_blank");
+    window.open("https://github.com/code-kern-ai/refinery-lf-exec-env/blob/dev/requirements.txt", "_blank");
```

Comment:
should this also link to lf exec or to record ide

Reply:
True, it should link to record ide